_contentTemplates/dropdowns/features.md

```diff
@@ -34,14 +34,18 @@ By virtualizing the elements in the dropdown, you can use huge data sources with
 
 | Parameter | Type | Description |
 | --- | --- | --- |
-| `AnimationDuration` | `int` | The animation duration of the popup in milliseconds. |
+| `AnimationDuration` | `int` <br /> (`300`) | Sets the dropdown animation duration in milliseconds. |
```

Comment:
The description of a parameter should not start with a verb. It should look like a big multi-word noun, except for bool parameters.

```suggestion
| `AnimationDuration` | `int` <br /> (`300`) | The dropdown animation duration in milliseconds. |
```